src/components/Rubric/hooks.ts

```diff
@@ -0,0 +1,98 @@
+import { useKeyedState, StrictDict } from '@edx/react-unit-test-utils';
+import { usePageData, useRubricConfig } from 'data/services/lms/hooks/selectors';
+import { submitRubric } from 'data/services/lms/hooks/actions';
+import { CriterionConfig, MutationStatus, RubricData } from 'data/services/lms/types';
+
+export type Criterion = {
```

Comment:
Love having types for the hooks, but I think I would rather we keep the convention of separating out types into a `types` module

Reply:
Should I make something like `types.ts`? This one is specific to the component. It was just nice when I was writing it in typescript. The props drilling go like `Pick<Criterion, 'name', 'description>`.

Reply:
yes.  v much `types.ts` per scope